src/app/components/Tab/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Tab } from "@headlessui/react";
+import { classNames } from "~/app/utils/index";
+
+type Props = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+  active: boolean;
+  icon?: React.ReactNode;
+  leftTab?: boolean;
+  rightTab?: boolean;
+  label?: string;
+};
+
```

Comment:
Maybe you could also export the other components from this tile we need for tabs, like Tab.Group and Tab.List, and we can apply any styling here if we need to.

Can you update the code in src/app/screens/Home/DefaultView/index.tsx to use the new tab components too?

Reply:
sure np